src/game.cpp

```diff
@@ -12307,7 +12307,7 @@ bool game::plmove(int dx, int dy)
             if (u.in_vehicle) {
                 m.unboard_vehicle(u.posx(), u.posy());
             }
-            u.power_level -= (tunneldist * 250); //tunneling costs 10 bionic power per impassable tile
+            u.charge_power(0 - (tunneldist * 250)); //tunneling costs 10 bionic power per impassable tile
```

Comment:
As you change this line anyway, please either remove the incorrect comment or correct it.